### DISCUSSION TITLE
Please add quora and google news as well

### DISCUSSION THREAD

**Discussion by theranajayant** [jordwest/news-feed-eradicator#398] [Ideas]
Please add quora and google news as well. 

URL 1: https://news.google.com/
URL 2: https://www.quora.com/

**Comment by ChillarAnand** [comment]
This seems to be an old request. Any plans to work on it soon?

**Comment by jordwest** [comment]
I'll be allowing new sites in v3 (or v3.1) when dynamic site lists are implemented, that will make it much easier to implement and maintain additional sites
